auto_route/example/pubspec.yaml

```diff
@@ -2,6 +2,9 @@ name: example
 description: A new Flutter application.
 version: 1.0.0+1
 
+publish_to: none
```

Comment:
By adding this property we won't be able to publish the package anymore. Is that what we really want?

Reply:
I recall linter complaining about this line otherwise, in retrospect I should have added it to ignore rule instead.